sanic/utils.py

```diff
@@ -11,35 +11,18 @@
 
 
 def str_to_bool(val: str) -> bool:
-    """Takes string and tries to turn it into bool as human would do.
-
-    If val is in case insensitive (
-        "y", "yes", "yep", "yup", "t",
-        "true", "on", "enable", "enabled", "1"
-    ) returns True.
-    If val is in case insensitive (
-        "n", "no", "f", "false", "off", "disable", "disabled", "0"
-    ) returns False.
-    Else Raise ValueError."""
-
-    val = val.lower()
-    if val in {
-        "y",
-        "yes",
-        "yep",
-        "yup",
```

Comment:
This is a breaking change. Why remove it?

Reply:
because it's not compliant with the functionality we're replacing, plus do people really use 'yep' and 'yup'? in every place we're actively using this it's only to check for 'no'

Reply:
... that we are using, yes. But it has been in the lib. There doesn't seem to be a compelling reason to remove this. And, if we did it would need to be done with deprecation. I'd say let's continue to support those because it already exists. The only people using "yup" would be someone that already expects it as a truthy value.